packages/@react-spectrum/radio/src/RadioGroup.tsx

```diff
@@ -94,6 +98,15 @@ function RadioGroup(props: SpectrumRadioGroupProps, ref: DOMRef<HTMLDivElement>)
           {children}
         </RadioContext.Provider>
       </div>
+      {hasHelpText &&
+        <HelpText
+          descriptionProps={descriptionProps}
+          errorMessageProps={errorMessageProps}
+          description={description}
+          errorMessage={errorMessage}
+          validationState={validationState}
+          showErrorIcon={showErrorIcon} />
```

Comment:
Same here as above, mind adding a story for error icon?

Reply:
added